Makefile

```diff
@@ -43,6 +43,11 @@ minikube-tunnel: # @HELP Open a tunnel to the local cluster and expose it on an
 minikube-tunnel:
 	./bin/minikube-tunnel
 
+.PHONY: argocd-port-forward
+argocd-port-forward: # @HELP Forward port 8080 of argocd-server to access it on the host system
+argocd-port-forward:
```

Comment:
suggestion: scope this to a specific environment. I'm not sure if this is intended to be for `local` `staging` or `production` or all of them. At the moment we are mostly keeping the `make` targets constant independent of what the user currently has their kubecontext set to

Reply:
*If* this is just for `local` why not add a special ingress like we do for mailhog?

Reply:
> If this is just for local why not add a special ingress like we do for mailhog?
I thought of that as well, but I haven't looked into it yet. Might be best to remove this port forwarding target alltogether